envpool/atari/atari_env.h

```diff
@@ -85,14 +85,15 @@ class AtariEnv : public Env<AtariEnvSpec> {
   const int kRawWidth = 160;
   const int kRawSize = kRawWidth * kRawHeight;
   std::unique_ptr<ale::ALEInterface> env_;
+  uint8_t arr[65536];
```

Comment:
I finally put it inside AtariEnv class because the palette initialization is called by env.loadROM(), which requires the base\_path in config to construct the rom path.

Reply:
arr is not a meaningful name.

Reply:
To avoid intialiazing it multiple times, we can make it a static member, and use a local static variable in the constructor to guarantee it is only initialized once.